src/collective/listingviews/tests/listingviews.rst

```diff
@@ -185,26 +187,38 @@ Note the html is in exactly the same order as we specifed in our view definition
 
 >>> print browser.contents
 <...
-    <dt class="listing-field field-Title">Title</dt>
-    <dd class="listing-field field-Title">item1</dd>
+                              <dt class="listing-field field-Title">Title</dt>
+                              <dd class="listing-field field-Title">item1</dd>
 ...
 
 >>> print browser.contents
 <...
-    <dt class="listing-field field-Title-tolink">Title (Link)</dt>
-    <dd class="listing-field field-Title-tolink"><a href="http://nohost/plone/folder1/item1">item1</a></dd>
+                              <dt class="listing-field field-Title-tolink">Title</dt>
+                              <dd class="listing-field field-Title-tolink"><a href="http://nohost/plone/folder1/item1">item1</a></dd>
 ...
 
 >>> print browser.contents
 <...
-    <dt class="listing-field field-...-localshort">Effective...Date (Date)</dt>
-    <dd class="listing-field field-...-localshort">Jan 01, 2001</dd>
+                              <dt class="listing-field field-lead_image-tagImage">lead_image</dt>
+                              <dd class="listing-field field-lead_image-tagImage"><img src="http://nohost/plone/folder1/item3/@@images/image" /></dd>
 ...
 
 >>> print browser.contents
 <...
-    <dt class="listing-field pubdate">Local Publication Date</dt>
-    <dd class="listing-field pubdate">01/01/2001</dd>
+                              <dt class="listing-field field-lead_image-tagImage">lead_image</dt>
+                              <dd class="listing-field field-lead_image-tagImage"><img src="http://nohost/plone/folder1/item4/@@images/image" /></dd>
```

Comment:
put more tests in the test_registrations for the different combinations like url, different scales etc

Reply:
need to test these the urls produced actually return an image for the different versions of plone. test this in test_registrations